src/test/java/IP/duke/main/DukeExceptionTest.java

```diff
@@ -0,0 +1,32 @@
+package IP.duke.main;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+/**
+ * Represents testing the DukeException class.
+ */
+public class DukeExceptionTest {
+    
+    /**
+     * Tests the getMessage method.
+     */
+    @Test
+    public void getMessage_StringOutOfBoundsException_CommandIncompleteMessage(){
+        StringIndexOutOfBoundsException e = new StringIndexOutOfBoundsException();
+        DukeException dukeException = new DukeException(e);
+        assertEquals(dukeException.getMessage(), "☹ The task description or command is incomplete.:$\n");
```

Comment:
Not a standard violation, but shouldn't the expected result be the first parameter?
```suggestion
        assertEquals("☹ The task description or command is incomplete.:$\n", dukeException.getMessage());
```